ios/Codeforces Watcher/Application/features/problems/ProblemsViewController.swift

```diff
@@ -27,7 +32,8 @@ class ProblemsViewController: UIViewController, StoreSubscriber, UISearchResults
 
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
-
+        
+        fabButton.isHidden = false
```

Comment:
`fabButton` is better to be added and controlled in `MainViewController`? Having separate buttons in each sub-controller complicates the code, no?